src/transformers/modeling_utils.py

```diff
@@ -808,7 +809,13 @@ def _load_state_dict_into_meta_model(
     for old_key, new_key in zip(old_keys, new_keys):
         state_dict[new_key] = state_dict.pop(old_key)
 
-    for param_name, param in state_dict.items():
+    # Show shard-level progress. Useful to monitor quantization progress
+    quant_show_progress = False
```

Comment:
Let's see what @amyeroberts think about this here!

Reply:
This shouldn't be in modeling_utils :) 

Monitoring quantization should be handled by the quantization objects, if at all

Reply:
Resolved since this progress bar was removed.